src/sentry/utils/security/orgauthtoken_token.py

```diff
@@ -9,8 +9,17 @@
 SENTRY_ORG_AUTH_TOKEN_PREFIX = "sntrys_"
 
 
+class SystemUrlPrefixMissingException(Exception):
+    "system.url-prefix is not set. You need to set this to generate a token."
+    pass
+
+
 def generate_token(org_slug: str, region_url: str):
     sentry_url = settings.SENTRY_OPTIONS.get("system.url-prefix")
```

Comment:
Any reason to not use `options.get("system.url-prefix")`? Most application uses `options.get` instead of indexing into `SENTRY_OPTIONS`. Accessing `SENTRY_OPTIONS` directly might be a contributing factor to this missing in self-hosted.

Reply:
TBH not that I know of, I _probably_ copied this from some other place. I'll replace it with `options.get`!